examples/ArduinoOTA/ArduinoOTA.ino

```diff
@@ -0,0 +1,52 @@
+/*
+    WifiSettings Arduino OTA example
+
+    Demonstrates how to run Arduino OTA in tandem with WifiSettings,
+    using the WifiSettings credentials.
+
+    Source and further documentation available at
+    https://github.com/Juerd/esp32-WiFiSettings
+*/
+
+#include <SPIFFS.h>
+#include <WiFi.h>
+#include <WiFiSettings.h>
+#include <ArduinoOTA.h>
+
+// Start ArduinoOTA via WiFiSettings with the same hostname and password
+void setup_ota() {
+    ArduinoOTA.setHostname(WiFiSettings.hostname.c_str());
+    ArduinoOTA.setPassword(WiFiSettings.password.c_str());
+    ArduinoOTA.begin();
+}
+
+void setup() {
+    Serial.begin(115200);
+    SPIFFS.begin(true);  // Will format on the first run after failing to mount
+
+    WiFiSettings.secure = true; // Force a secure portal
+
+    // Set callbacks to start OTA when the portal is active
+    WiFiSettings.onPortal = []() {
+        setup_ota();
+    };
+    WiFiSettings.onPortalWaitLoop = []() {
+        ArduinoOTA.handle();
+    };
+
+    Serial.print("Password: ");
```

Comment:
.portal() that is called from .connect() already prints the password. If printing the password is moved to *after* .connect(), then it is only printed once, and only if the portal is not started.

Reply:
As intended. If the WiFi settings are already saved by a sketch where the portal wasn't secured, the user would have no straight-forward way of knowing the password which is now necessary for OTA updates.

Reply:
What's the rationale for printing the password *twice* when the connection fails and the portal is started?

Reply:
My mistake, I thought I'd already placed it after `.connect()`. Fixed in the latest commit.